core/src/main/java/com/linecorp/armeria/common/QueryParamGetters.java

```diff
@@ -95,7 +96,8 @@ interface QueryParamGetters extends StringMultimapGetters</* IN_NAME */ String,
      *
      * @param name the parameter name
      * @return the {@code boolean} value of the first value in insertion order or {@code null}
-     *         if there is no such parameter or it can't be converted to {@code boolean}.
+     *         if there is no such parameter or it can't be converted to {@code boolean}
+     *         by {@link StringUtil#toBoolean}.
```

Comment:
`StringUtil#toBoolean` is an internal API that can't be exposed in Javadoc.

Reply:
✅ updated.

Aha~ `StringUtil#toBoolean` is an internal API! I miss it 😅
I removed `StringUtil#toBoolean` on comment and just explained it's behavior ~!